modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -50,6 +52,10 @@ class UploadSubmission < ApplicationRecord
 
     scope :for_consumer, ->(consumer) { where(consumer_name: consumer) }
 
+    scope :not_from_appeals_api, lambda {
+      where("consumer_name IS NULL OR consumer_name NOT ILIKE '%#{APPEALS_API_CONSUMER_PATTERN}%'")
+    }
```

Comment:
Used to exclude evidence submissions from certain Benefits Intake alerts and reporting.